core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/CredentialBase.java

```diff
@@ -81,7 +81,6 @@ public String toString() {
         return "CredentialBase{"
                 + "name='" + name + '\''
                 + ", cloudPlatform='" + cloudPlatform + '\''
-                + ", parameters=" + parameters
```

Comment:
please replace it with something like this **parameters=*****, instead of deleting it